README.md

```diff
@@ -276,6 +276,36 @@ public static void DocumentClient(
 }
 ```
 
+###Azure Notification Hubs
+
+An [Azure Notification Hub](https://azure.microsoft.com/en-us/services/notification-hubs/) binding allows you to easily send push notifications to any platform. This extension lives in **Microsoft.Azure.WebJobs.Extensions.NotificationHubs** nuget package. To configure the binding, add the NotificationHubs namespace connection string as an app setting or environment variable using the setting name `AzureWebJobsNotificationHubsConnectionString` and add the name of the NotificationHub as an app setting or environment variable using the setting name `AzureWebJobsNotificationHubName`.
+
+Azure Notification Hub must be configured for the Platform Notifications Services (PNS) you want to use. For more information on configuring an Azure Notification Hub and developing a client applications that register for notifications, see [Getting started with Notification Hubs] (https://azure.microsoft.com/en-us/documentation/articles/notification-hubs-windows-store-dotnet-get-started/) and click your target client platform at the top.
+
+The following sample sends windows toast notification on a TimerTrigger
+
+```csharp
+ public static void SendWindowsNotification(
+    [TimerTrigger("*/30 * * * * *")] TimerInfo timerInfo,
```

Comment:
On the main readme page it would be best if you had a real world example. Nobody is going to set up a timer to send notifications :)